client/js/components/month/month-day-event.js

```diff
@@ -0,0 +1,23 @@
+/** @jsx React.DOM */
+var React = require('react');
+var Link = require('react-router-component').Link;
+var moment = require('moment');
+var AppViewActions = require('../../actions/view-action-creator');
+
+var MonthDayEvent = React.createClass({
+  handleClick: function () {
+    var date = moment(this.props.eventData.date, 'YYYY-MM-DD');
+    AppViewActions.getDayData(date);
+  },
+  render: function () {
```

Comment:
This is a bit repetitive. Maybe something like:

var color = this.props.health === 10 ? 'green' : this.props.health < 4 ? 'red' : 'blue';
return <div onClick={ this.handleClick } className={ 'events-circle' + color } ></div>;